src/macros.jl

```diff
@@ -188,6 +188,15 @@ function parse_constraint(_error::Function, sense::Symbol, lhs, rhs)
     (sense, vectorized) = _check_vectorized(sense)
     vectorized, parse_one_operator_constraint(_error, vectorized, Val(sense), lhs, rhs)...
 end
+function parse_constraint(_error::Function, ::Val{:call}, args...)
+    return parse_constraint(_error, Val(args[1]), args[2:end]...)
+end
+
+function parse_constraint(_error::Function, sense::Val, F...)
```

Comment:
Why are these two methods needed ?

Reply:
Indeed, they are not needed, I've just removed them.